ansible_collections/arista/avd/roles/eos_cli_config_gen/README.md

```diff
@@ -1796,6 +1797,19 @@ management_ssh:
   log_level: < SSH daemon log level >
 ```
 
+#### Management Tech-Support
+
+```yaml
+management_tech_support:
+  policy_show_tech_support:
+    exclude_commands:
+      - command: < command_to_exclude >
+        type: < text | json | default -> text >
```

Comment:
Please add a comment above this line 'The supported values for type are platform dependent'. Add comment at same indentation level.